packages/kbn-pm/dist/index.js

```diff
@@ -207,7 +207,7 @@ async function run(argv) {
       cache: true,
       validate: true
     },
-    boolean: ['prefer-offline', 'frozen-lockfile', 'cache', 'validate']
```

Comment:
It's only happened like twice in my life, and I question the next time I'll be on an airplane, but I definitely appreciated being able to choose `--prefer-offline` in the past, and needing to customize it on a file that's tracked by git seems unnecessary. Why get rid of the option for people to specify it?

Reply:
We have been using prefer-offline on by default through the `.yarnrc` file configuration and it could be managed using it. Do you think it works that way? After talking with @tylersmalley we talked about remove this as keep supporting it through the `kbn bootstrap` imply we call `bazel run @nodejs//:yarn -- -...options` before running `bazel build ...` which we are trying to avoid.

Reply:
Ah, forgot that we turned it on by default 😅